migrations/20170309102759_hot_score_as_float.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function(knex, Promise) {
```

Comment:
FYI, no need to change if you don't want to: since you aren't using the Promise-variable, you can drop it from the function arguments